docs/devel/local-cluster/docker.md

```diff
@@ -225,6 +214,10 @@ On OS X, since docker is running inside a VM, run the following command instead:
 docker-machine ssh `docker-machine active` curl $ip
 ```
 
+## Deploy a DNS
+
+Read [documentation for manually deploying a DNS](http://kubernetes.io/docs/getting-started-guides/docker-multinode/#deploy-dns-manually-for-v12x) for instructions.
```

Comment:
I am not sure the policy for cross linking.  We should probably develop a story around that with devrel.  These docs are generally in a bad state and we are moving them out of the docs repo to reduce maintenance overhead.  I am less concerned about the links breaking than I am about the staleness of the technical content (At least when the links break it is obvious what is broken). I am just porting over this change because it was made in the docs repo before after I moved it to this repo.